crypto/keys/src/x25519.rs

```diff
@@ -45,22 +49,25 @@ impl AsRef<[u8]> for X25519Secret {
 
 /// The debug implementation will output the SHA-256 sum of the secret, but not
 /// the secret itself
-impl Debug for X25519Secret {
-    fn fmt(&self, f: &mut Formatter) -> FmtResult {
+impl core::fmt::Debug for X25519Secret {
+    fn fmt(&self, f: &mut core::fmt::Formatter) -> core::fmt::Result {
```

Comment:
Nit: Unnecessary prefix
```suggestion
impl Debug for X25519Secret {
    fn fmt(&self, f: &mut Formatter) -> FmtResult {
```